masterlist.yaml

```diff
@@ -37,6 +37,16 @@ common:
         text: "Du ser ud til at bruge %1%, men du bruger ikke “%1%”-lappen for dette plugin: %2%"
       - lang: ja
         text: "%1%が使用されていますが、このプラグイン用の%1%のパッチが有効になっていません: %2%"
+  - &hasPatchIncluded
+    type: warn
+    content:
+      - lang: en
+        text: "A patch is included with the %2% installer, please make sure you have installed the %1% patch and have it activated"
+  - &hascompatibilityIssues
+    type: say
+    content:
+      - lang: en
+        text: '%1% is not fully compatibility with "%2%".%3%'
```

Comment:
For cases where two plug-ins are not incompatible, but still have conflicts that can cause some issues with NPC pathing or visual bugs.